model/src/main/proto/javaclasses/exlibris/c/events.proto

```diff
@@ -347,9 +351,9 @@ message BookRemoved {
 
     // The reason of removing book.
     //
-    // Book could be remove with one of the next reason:
+    // Book could be removed with one of the next reason:
```

Comment:
The book could be removed with one of the following reasons:
or  The book could be removed with one of the reasons below: